src/views/refund/steps.js

```diff
@@ -28,78 +31,69 @@ const stepOneStyles = theme => ({
   },
 });
 
-const StyledStepOne = ({ classes }) => (
+const StyledStepOne = ({ classes, setRefundFile, setTransactionHash }) => (
   <View className={classes.wrapper}>
-    <DropZone className={classes.dropZone}>
+    <DropZone className={classes.dropZone} onFileRead={setRefundFile}>
```

Comment:
Do you think we should add some type of visual indicator that the file has been uploaded? Not necessary for this PR but we should consider it.

Reply:
I don't think we have to do that in this PR but I will open an issue for it.